vpr/src/pack/cluster.cpp

```diff
@@ -2008,92 +2044,92 @@ void add_cluster_molecule_candidates_by_connectivity_and_timing(t_pb* cur_pb,
+                        add_molecule_to_pb_stats_candidates(molecule,
+                                cur_pb->pb_stats->gain, cur_pb, AAPACK_MAX_FEASIBLE_BLOCK_ARRAY_SIZE);
                     }
                 }
-			}
-		}
-	}
+            }
+        }
+    }
+}
 
 void add_cluster_molecule_candidates_by_highfanout_connectivity(t_pb* cur_pb,
                                                                 t_cluster_placement_stats *cluster_placement_stats_ptr,
                                                                 const std::multimap<AtomBlockId,t_pack_molecule*> &atom_molecules) {
-		/* Because the packer ignores high fanout nets when marking what blocks
-         * to consider, use one of the ignored high fanout net to fill up lightly
-         * related blocks */
-		reset_tried_but_unused_cluster_placements(cluster_placement_stats_ptr);
+    /* Because the packer ignores high fanout nets when marking what blocks
+     * to consider, use one of the ignored high fanout net to fill up lightly
+     * related blocks */
+    reset_tried_but_unused_cluster_placements(cluster_placement_stats_ptr);
 
-		AtomNetId net_id = cur_pb->pb_stats->tie_break_high_fanout_net;
+    AtomNetId net_id = cur_pb->pb_stats->tie_break_high_fanout_net;
 
     auto& atom_ctx = g_vpr_ctx.atom();
 
     int count = 0;
-        for(auto pin_id : atom_ctx.nlist.net_pins(net_id)) {
-            if(count >= AAPACK_MAX_HIGH_FANOUT_EXPLORE) {
-                break;
-            }
+    for(auto pin_id : atom_ctx.nlist.net_pins(net_id)) {
+        if(count >= AAPACK_MAX_HIGH_FANOUT_EXPLORE) {
+            break;
+        }
 
-            AtomBlockId blk_id = atom_ctx.nlist.pin_block(pin_id);
+        AtomBlockId blk_id = atom_ctx.nlist.pin_block(pin_id);
 
-			if (atom_ctx.lookup.atom_clb(blk_id) == ClusterBlockId::INVALID()) {
+        if (atom_ctx.lookup.atom_clb(blk_id) == ClusterBlockId::INVALID()) {
 
-                auto rng = atom_molecules.equal_range(blk_id);
-                for(const auto& kv : vtr::make_range(rng.first, rng.second)) {
+            auto rng = atom_molecules.equal_range(blk_id);
+            for(const auto& kv : vtr::make_range(rng.first, rng.second)) {
                 t_pack_molecule* molecule = kv.second;
-					if (molecule->valid) {
+                if (molecule->valid) {
                     bool success = true;
                     for (int j = 0; j < get_array_size_of_molecule(molecule); j++) {
-							if (molecule->atom_block_ids[j]) {
-								VTR_ASSERT(atom_ctx.lookup.atom_clb(molecule->atom_block_ids[j]) == ClusterBlockId::INVALID());
-								auto blk_id2 = molecule->atom_block_ids[j];
-								if (!exists_free_primitive_for_atom_block(cluster_placement_stats_ptr, blk_id2)) {
-                                    /* TODO: debating whether to check if placement exists for molecule (more
-                                     * robust) or individual atom blocks (faster) */
-									success = false;
-									break;
```

Comment:
Can you remove changes that are purely white space changes?

Reply:
Done